mlflow/server/prometheus_exporter.py

```diff
@@ -1,32 +1,19 @@
 from prometheus_flask_exporter.multiprocess import GunicornInternalPrometheusMetrics
 from flask import request
 
+from mlflow.version import VERSION
+
 
 def activate_prometheus_exporter(app):
-    metrics = GunicornInternalPrometheusMetrics(app, export_defaults=False)
+    def mlflow_version(req: request):  # pylint: disable=unused-argument
+        return VERSION

```

Comment:
```suggestion
    def mlflow_version(_: request):
        return VERSION

```

Can we use `_` here?